src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -56,11 +59,33 @@
       ;; Respond to all CSR validation failures with a 400
       (middleware-utils/plain-response 400 msg))))
 
-(defn handle-get-certificate-revocation-list
-  [{:keys [cacrl]}]
-  (-> (ca/get-certificate-revocation-list cacrl)
-      (rr/response)
-      (rr/content-type "text/plain")))
+  (schema/defn format-http-date :- (schema/maybe org.joda.time.DateTime)
+    "Formats an http-date into joda time.  Returns nil for malformed or nil http-dates"
+    [http-date :- (schema/maybe schema/Str)]
+    (if http-date
+      (try
+        (time-format/parse
+          (time-format/formatters :rfc822)
+          (string/replace http-date #"GMT" "+0000"))
+        (catch IllegalArgumentException e
+          nil))
+      nil))
+
+  (defn handle-get-certificate-revocation-list
+    "Always return the crl if no 'if-modified-since' header is provided or if that header is not in correct http-date format.
```

Comment:
Can you wrap these lines a little earlier. I think most docs in this file/repo try to keep the wrap around at 80 ish characters.

Reply:
Might want to capitalize 'if-modified-since' as we do below

Reply:
Instead of `in correct http-date format` we also might just want to call out `rfc822`

Reply:
Wow, those were super nit-picky of me, sorry.

Reply:
The only thing I didnt change was http-date to rfc822, since our http date has to end in GMT, which is not the case for rfc822 i believe. (rfc822 <pubDate>Wed, 02 Oct 2002 15:00:00 +0200</pubDate> is OK, for example, but wouldnt be a valid http-date since it needs to end in GMT :)

Reply:
That's fair